libbeat/etc/kibana/dashboard.json.j2

```diff
@@ -0,0 +1,13 @@
+{
+  "hits": 0,
+  "timeRestore": false,
+  "description": "",
+  "title": "{{beatname}} Dashboard",
```

Comment:
I would avoid having a template for each dashboard, visualization, search, index-pattern as the format might change from version to version in Kibana.